src/NodaTime.Test/InstantTest.Operators.cs

```diff
@@ -81,6 +82,12 @@ public void OperatorPlusOffset_Zero_IsNeutralElement()
             Assert.AreEqual(new LocalInstant(0, 1L), one.Plus(Offset.Zero), "Instant(1) + Offset.Zero");
             Assert.AreEqual(new LocalInstant(0, NodaConstants.NanosecondsPerHour), NodaConstants.UnixEpoch.Plus(Offset.FromHours(1)), "UnixEpoch + offsetOneHour");
         }
+
+        [Test]
+        public void OperatorPlus_OutOfRange()
+        {
+            Assert.Throws<OverflowException>(() => (Instant.MaxValue + Duration.Epsilon).GetHashCode());
```

Comment:
Obvious answer, I'm sure, but why the `GetHashCode()`?

Reply:
Because `Instance.MaxValue + Duration.Epsilon` isn't a statement expression, so the lambda expression can't be converted to `Action`. (It's a shame `Assert.Throws` doesn't have an overload accepting a `Func<T>`.)